src/components/ChattyLLM/ChattyLLMInputForm.vue

```diff
@@ -248,6 +251,7 @@ export default {
 			editingTitle: false,
 			pollMessageGenerationTimerId: null,
 			pollTitleGenerationTimerId: null,
+			autoplayAudioChat: loadState('assistant', 'autoplayAudioChat', true),
```

Comment:
should be `autoplay_audio_chat`
and `audio_chat_available` should be fetched here too to not show the mic icon.